src/wasm/openmp.cpp

```diff
@@ -581,11 +587,18 @@ namespace wasm {
      */
     WAVM_DEFINE_INTRINSIC_FUNCTION(env, "__kmpc_reduce", I32, __kmpc_reduce, I32 loc, I32 gtid, I32 num_vars,
                                    I32 reduce_size, I32 reduce_data, I32 reduce_func, I32 lck) {
-
+        throw std::runtime_error("RIP");
```

Comment:
Worth logging before throwing the exception to know what's killed it (same applies to other dead methods in here)

Reply:
Woops those are still meant to run if not distributed!
I need a way to elegantly handle the "am I in a distributed Level" paradigm. I was planning to do this in a later MR by extending the `OMPLevel` class into two subclasses and re-implement those functions in that. Because that would bloat that MR